dip-0003/network-info.md

```diff
@@ -0,0 +1,43 @@
+# Appendix C: Network Information
+
+## Masternode Information
+
+The fields below are represented as `networkInfo` and are applicable to all masternodes types as defined in
+[Appendix B](masternode-types.md)
+
+| Field     | Type    | Size | Description                                                                |
+| --------- | ------- | ---- | -------------------------------------------------------------------------- |
+| ipAddress | byte[]  | 16   | IPv6 address in network byte order. Only IPv4 mapped addresses are allowed |
+| port      | uint_16 | 2    | Port (network byte order)                                                  |
+
+### <a name="mninfo_rules">Validation Rules</a>
+
+* `ipAddress` MUST be a valid IPv4 address that is routable on the global internet
+* `ipAddress` MUST NOT be already used in the registered masternodes set
+* `port` MUST be within the valid port range [1, 65535]
+* On mainnet, `port` MUST be set to the following value
+
+  | Field  | Value  |
+  | -------| ------ |
+  | port   | 9999   |
+
+## Platform Information
+
+The fields below are represented as `platformInfo` and are only applicable for EvoNodes as defined in
```

Comment:
I feel we should refer to all as something like platformNetInfo
```suggestion
The fields below are represented as `platformNetInfo` and are only applicable for EvoNodes as defined in
```

Reply:
Resolved in latest push